blog-homepage/src/modules/yourArticles.jsx

```diff
@@ -0,0 +1,105 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+// Data
+import your from "../_data/your-articles.json";
+// my Components
+import Picture from "./pictures";
+import DateBuild from "./dateBuild";
+
+
+// BootStrap Components
+import Card from "react-bootstrap/Card";
+import Button from "react-bootstrap/Button";
+// import MissedArticles from "./dArticles";
+
+
+const YourArticles = ()=>{
+  // check if preview exist
+  const preview = (isPreview, hasAudioAvailable)=>{
+    if(isPreview===true && hasAudioAvailable===true){
+      return "🔊 Audio Available ⭐";
+    }
+    if(isPreview===true){
+      return "⭐ Member Preview";
+    }else{
+      return "";
+    }
+  };
+  // check if is author member
+  const member = (isMember)=>{
+    if(isMember===true){
+      return "forYouTitleImgMember";
+    }else{
+      return "forYouAuthorImg";
+    }
+  };
+  const buttonID = (id)=>{
+    if(id>=0){
+      return `forYouButtonID_${id}`;
+    }
+  };
+  const pictureID= (id)=>{
+    if(id>=0){
+      return `forYouAuthorPictureID_${id}`;
+    }
+  };
+  const nameID =(id)=>{
+    if(id>=0){
+      return `forYouAuthorNameID_${id}`;
+    }
+  };
+  const dateID = (id)=>{
+    if(id>=0){
+      return `forYouPostedDateID_${id}`;
+    }
+  };
+  const minID = (id)=>{
+    if(id>=0){
+      return `forYouMinToReadID_${id}`;
+    }
+  };
+  const addBookMark = (value)=>{
+    let article = your[value].title;
+    let articleAuthor = your[value].author.name;
+    alert(`New Book Mark: 
+      ${article}
+      by: ${articleAuthor} `);
+  };
+  const titleID = (id)=>{
+    return `forYouArticleTitleID_${id}`;
+  };
+  const cardID = (id)=>{
+    if(id>=0){
+      return`forYouCardID_${id}`;
+    }
+  };
+  const descriptionID = (id)=>{
+    return `forYouArticleDescriptionID_${id}`;
+  };
+  return(
+    <div className="forYouArticles">
+    {
+      your.map((article, index)=>(
+        <Card className="forYouCard" id={cardID(index)} key={index}>
```

Comment:
Note that this code is pretty similar to the missed articles component. This could be a good opportunity to create an "article" component in order to avoid duplication.